llms/mixtral/mixtral.py

```diff
@@ -76,7 +56,7 @@ def __init__(self, args: ModelArgs):
         self.wk = nn.Linear(args.dim, args.n_kv_heads * args.head_dim, bias=False)
         self.wv = nn.Linear(args.dim, args.n_kv_heads * args.head_dim, bias=False)
         self.wo = nn.Linear(args.n_heads * args.head_dim, args.dim, bias=False)
-        self.rope = RoPE(args.head_dim, traditional=True)
+        self.rope = nn.RoPE(args.head_dim, traditional=True)
```

Comment:
In order for this to match the above we should probably pass the base parameter to RoPE. ie `nn.RoPE(args.head_dim, base=1000000, traditional=True)`.

Reply:
It was working well even with base Mistral value 🤔

Reply:
Hm interesting. Let me go search which one should be there and then I 'll merge this. Thanks!

Reply:
1000000 is the right one, but please use #470 from @jbrodriguez